pkgs/top-level/release.nix

```diff
@@ -54,43 +57,36 @@ let
               jobs.manual
               jobs.lib-tests
               jobs.stdenv.x86_64-linux
-              jobs.stdenv.i686-linux
```

Comment:
I know I'm late, but I'd personally keep stdenv in there, as we still want to support *some* packages built by it (e.g. wine).